File: 백준/Bronze/2530. 인공지능 시계/인공지능 시계.py
a, b, c = map(int, input().split())
d = int(input())

hour = a + d // 3600
d %= 3600

minute = b + d // 60
d %= 60

second = c + d

if second >= 60:
    minute += second // 60
    second %= 60

if minute >= 60:
    hour += minute // 60
    minute %= 60

hour %= 24

print(hour, minute, second)

File: 백준/Bronze/2530. 인공지능 시계/인공지능 시계.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String[] str = sc.nextLine().split(" ");
        int a = Integer.parseInt(str[0]);
        int b = Integer.parseInt(str[1]);
        int c = Integer.parseInt(str[2]);

        int d = sc.nextInt();

        int hour, minute, second;

        hour = (d / 3600) + a;
        d %= 3600;

        minute = (d / 60) + b;
        d %= 60;

        second = (d + c);

        if (second > 59)
        {
            second -= 60;
            minute++;
        }

        if (minute > 59)
        {
            minute -= 60;
            hour++;
        }

        hour %= 24;

        System.out.println(hour + " " + minute + " " + second);
    }
}